### PR TITLE
Add account page on the explorer

### PR DESCRIPTION
**Pull request by antho1404**
Add the `/address/xxx` page to check the balance and list of transactions of a specific address.
All attributes that match the address format with a link to this page (example transaction that has a send message).

Screen Shot 2020-03-03 at 17 45 18

### CODE REVIEW

File: components/Attribute.vue [hunk 1]
```diff
@@ -16,6 +19,13 @@ export default {
     },
     value() {
       return this.decode(this.attribute.value)
+    },
+    to() {
+      const addressRegexp = new RegExp(`^${BECH32_PREFIX}[a-z0-9]*$`)
```
Comment (NicolasMahe) [inline components/Attribute.vue]: The regex should be improved. It currently matches addresses like: `mesgtestvaloper1mzmyqx0lr6nxj0lu4czqkgs3daa94wpeuayrcn`.
The length of the addresses is exactly 39 characters after the prefix.


```suggestion
      const addressRegexp = new RegExp(`^${BECH32_PREFIX}[a-z0-9]{39}$`)
```

Comment (NicolasMahe) [reply components/Attribute.vue]: Getting the balance of `mesgtestvaloperXXX` returns an error 500:
http://localhost:3000/address/mesgtestvaloper1mzmyqx0lr6nxj0lu4czqkgs3daa94wpeuayrcn

**Review by NicolasMahe — APPROVED** [review]
except the suggestion I put, I'm good to merge it

File: components/Attribute.vue (hunk 1 above)
Comment (antho1404) [reply components/Attribute.vue]: done

**Pull request merged**